framework/lib/components/combo-picker/combo-picker.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react'
+import { Box, Button, InputGroup, InputRightElement, NumberInputField, useDisclosure } from '@chakra-ui/react'
```

Comment:
Moved Box as a Northlight import, about the rest I need them from Chakra since in Northlight they force me to use Labels and names when, this component being a wrapper, I don't want to give them to the inner components

Reply:
Hey, yeah that is fine, Northlight NumberInputField is different from chakra, but are you sure you changed the import for box and button? It should look like: 
![Screenshot 2024-09-24 at 16 21 44](https://github.com/user-attachments/assets/ef6a661a-7fb3-4075-b8a2-54d23c872197) but it still looks like you import from chakra?